sys-apps/policycoreutils/policycoreutils-2.4.ebuild

```diff
@@ -100,6 +100,7 @@ src_compile() {
 			INOTIFYH="$(usex dbus)" \
 			SESANDBOX="n" \
 			CC="$(tc-getCC)" \
+			PREFIX="${ROOT}" \
```

Comment:
is this prefix value only significant during build? prefix generally means the runtime path (such as /usr) so if it influences runtime there could be problems.